infra/gcp_service_account/main.tf

```diff
@@ -20,12 +20,6 @@ resource "google_project_iam_member" "cloudruntimeconfiguratoradmin" {
   member  = "serviceAccount:${google_service_account.dss.email}"
 }
 
-resource "google_project_iam_member" "storageadmin" {
-  project = "${data.google_project.project.project_id}"
-  role    = "roles/storage.admin"
-  member  = "serviceAccount:${google_service_account.dss.email}"
-}
-
```

Comment:
Consider separating `resource "google_project_iam_member" "storageadmin"` removal into it's own PR to make it prominent in the repo history.